dtmi/com/hms_networks/ewon_flexy_azure_connector_basic-1.json

```diff
@@ -0,0 +1,19 @@
+{
+	"@context": "dtmi:dtdl:context;2",
+	"@id": "dtmi:com:hms_networks:Ewon_Flexy_Azure_Connector_Basic;1",
+	"@type": "Interface",
+	"displayName": "Ewon_Flexy_Azure_Connector_Basic",
+	"contents": [
+		{
+			"@type": "Property",
+			"name": "DeviceMode",
+			"writable": true,
+			"schema": "string"
```

Comment:
suggestion. Have you considered using an `Enum` to describe the expected values for DeviceMode ?

Reply:
I read the documentation for `Enum`s and have updated the blank and basic models accordingly. There are only two possible values, the string "0" and string "1". Thank you for the suggestion.